test/test_torch.py

```diff
@@ -793,9 +793,27 @@ def _testSelection(self, torchfn, mathfn):
     def test_max(self):
         self._testSelection(torch.max, max)
 
+    @staticmethod
+    def _test_max_with_inf(self, device='cpu'):
+        a = torch.tensor([[-inf, -inf, inf, 3], [inf, inf, -inf, -1]], device=device)
```

Comment:
Is `inf` defined somewhere?

Reply:
torch._six :) see the import at top of the file.

Reply:
Oh oops, I missed that.